fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_coffin.dm

```diff
@@ -40,17 +40,17 @@
 
 /obj/structure/closet/crate/coffin/securecoffin
 	name = "secure coffin"
-	desc = "For those too scared of having their place of rest disturbed."
+	desc = "For insecure suckers to safely slumber in their secure sleeping space."
```

Comment:
```suggestion
	desc = "As a being of the undead, when you're caught claiming a life insurance contract for a dozenth time, no other coffin will provide an experience as secure as this one when it comes to protecting against the space IRS."
```

Reply:
improvement to your cringe line

Reply:
Too long, not changing it unless its a reference to something (or funny), try again nerd.

Reply:
fuck you

Reply:
literally googling "vampire puns" wouldve given you a better result than whatever you came up with